tools/gce/linux_performance_worker_init.sh

```diff
@@ -117,18 +122,25 @@ sudo apt-get update
 sudo apt-get install -y mono-devel nuget
 
 # C# .NET Core dependencies (https://www.microsoft.com/net/core#ubuntu)
-sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ xenial main" > /etc/apt/sources.list.d/dotnetdev.list'
+sudo sh -c 'echo "deb [arch=amd64] https://apt-mo.trafficmanager.net/repos/dotnet-release/ yakkety main" > /etc/apt/sources.list.d/dotnetdev.list'
 sudo apt-key adv --keyserver apt-mo.trafficmanager.net --recv-keys 417A0893
 sudo apt-get update
-sudo apt-get install -y dotnet-dev-1.0.0-preview2-003131
+sudo apt-get install -y dotnet-dev-1.0.0-preview2.1-003155
```

Comment:
That is problematic because whenever sdk version is set in global.json,  the SDK version needs to match exactly.
Since we migrated to the new .csproj projects recently, we are not using global.json in upstream/master or v1.3.x (so that's fine), but global.json is still in use in v1.2.x: https://github.com/grpc/grpc/blob/v1.2.x/src/csharp/global.json

Updating the workers would mean breaking the performance benchmarks for v1.2.x (our performance_released dashboard), since C# won't build.  On the other hand, we are hopefully a few days away from releasing v1.3.x and when that happens, we will also bump performance_release dashboard to v1.3.x and unbreak it.